consumer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM amazoncorretto:11-alpine-jdk
+COPY target/consumer-0.0.1-SNAPSHOT.jar app.jar
```

Comment:
To avoid depending on the version of the artifact when running the application via docker-compose, you can use a regular expression to denote the .jar file name.